### PR TITLE
i've made many changes 🤓

### PR DESCRIPTION
**Pull request by mutasim77**
I have made many changes. I created an MVP that we can now use. Check it out and let me know if you have any questions or suggestions. I'm here >>

### CODE REVIEW

File: base/models.py [hunk 1]
```diff
@@ -13,7 +14,7 @@
 
 """
```
Comment (BushraAks) [inline base/models.py:15]: aa so good you gave explanation

File: base/static/base/js/script.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+console.log('Hello World!');
+
+function validate() {
+
+    const author = document.querySelector('[name="author"]'),
+        content = document.querySelector('[name="content"]'),
+        image = document.querySelector('[name="image"]');
+
+    if (!author.value || !content.value || !image.value) {
+        console.log('Error'); // we need to show this error to user;
```
Comment (BushraAks) [inline base/static/base/js/script.js:10]: got it

**Review by BushraAks — APPROVED** [review]

File: base/models.py [hunk 3]
```diff
@@ -13,7 +14,7 @@
```
Comment (BushraAks) [inline base/models.py:14]: aa so good you showed explanation

**Review by BushraAks — APPROVED** [review]

File: base/static/base/js/script.js (hunk 2 above)
Comment (mutasim77) [reply base/static/base/js/script.js:10]: Genius

**Pull request merged**